pages/best-practices/avo-and-git.mdx

```diff
@@ -14,8 +14,9 @@ When changes are made to a tracking plan in Avo:
 1. Open a branch in Avo and make your changes
 2. Share the Avo branch with a developer
 3. Implement the changes in the Avo branch on a git branch. _We recommend having one git branch per source for each Avo branch_
-4. Merge the git branch
-5. Merge the Avo branch
+4. Before merging the git branch, make sure the git branch is up to date with the Avo branch. This can be done by running `avo pull`.
```

Comment:
Could this be done automatically via PR checks? Or are we missing the token thingy to be able to do that?

Reply:
We are missing token based API access in order to do that yes – soon tough!